modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -480,6 +489,45 @@ public String getImageURL(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		return cpMedia.getURL();
 	}
 
+	public int getMiniumOrderQuantity(long cpDefinitionId) {
+		CPDefinitionInventory cpDefinitionInventory =
+			_cpDefinitionInventoryLocalService.
+				fetchCPDefinitionInventoryByCPDefinitionId(cpDefinitionId);
+
+		return cpDefinitionInventory.getMinOrderQuantity();
+	}
+
+	@Override
+	public List<CommerceInventoryReplenishmentItem> getOrderedIncomingQuantity(
+		HttpServletRequest httpServletRequest, String sku) {
+
+		List<CommerceInventoryReplenishmentItem>
```

Comment:
Can you add a new method with the comparator in signature so that the returned list is already sorted?

Reply:
Are you sure this request was implemented?

Reply:
yes I've created the method with the comparator, and since the old method (that I've created before) is no longer required I've removed it